src/app/views/apps/modals/app-settings-modal.less

```diff
@@ -0,0 +1,58 @@
+//=======================================
+// Settings Modals
+//=======================================
+#app-settings-modal {
+
+  .modal-body {
+    padding: 5px;
+
+    div.app-name {
+      border: 0px;
+
+      .fa {
+        margin-left: 5px;
+      }
+    }
+
+    h4, .title {
+      text-transform: uppercase;
+      font-weight: bold;
+      font-size: 14px;
+    }
+
+    .sentence-check {
+      width: 100%;
+    }
+
+    .buttons {
+      width: 100%;
+      text-align: right;
+      padding-top: 20px;
+    }
+
+    .label {
+      font-size: 100%;
+      text-transform: capitalize;
+      padding: 6px 12px;
+
+      &.updating, &.upgrading, &.provisioning {
+        background-color: @fluroorange;
+      }
+      &.running, &.starting, &.restarting {
+        background-color: @darkgreen;
+      }
+      &.terminating, &.terminated {
+        background-color: @pink;
+      }
+      &.neutral {
+        background-color: @lightblue;
```

Comment:
Those colors should be less variables in `variables.less`. Also be careful, as thos colors are only defined in impac-angular. If impac-angular changes its colors variables this would break.